security/cmd/istio_ca/main.go

```diff
@@ -220,11 +238,22 @@ func runCA() {
 
 	verifyCommandLineOptions()
 
+	var webhooks map[string]controller.WebhookEntry
+	if opts.appendDNSNames {
+		webhooks = make(map[string]controller.WebhookEntry)
+		for i, svcAccount := range webhookServiceAccounts {
+			webhooks[svcAccount] = controller.WebhookEntry{
+				ServiceName: webhookServiceNames[i],
+				Namespace:   opts.istioCaStorageNamespace,
```

Comment:
This might work for now, but it is not scalable :(
In the key of webhookServiceAccounts, can we use a format "serviceaccount.namespace"?  We just need to parse it here and put the namespace part into the WebhookEnvoy. What do you think?